src/api/index.js

```diff
@@ -0,0 +1,75 @@
+const BASE = 'http://localhost:3000'
```

Comment:
This isn't used, so lets remove this before we merge it, and add it back when you are ready